integration/testdata/kritis-server/kritis-config.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kritis.grafeas.io/v1beta1
+kind: KritisConfig
+metadata:
+  name: my-kritis-config
+  namespace: default
```

Comment:
I think this may be the bug -- the file specifies the namespace as "default" but the integration tests try to deploy to a different namespace each time

Reply:
From the discussion offline, several of the namespace-related issues were fixed. Thanks!